e2e_tests/tests/deploy/test_local.py

```diff
@@ -243,7 +253,10 @@ def test_master_up_down() -> None:
     master_name = f"{cluster_name}_determined-master_1"
 
     with resource_manager(
-        Resource.MASTER, master_name, {"initial-user-password": conf.USER_PASSWORD}
+        Resource.MASTER,
+        master_name,
+        {"initial-user-password": conf.USER_PASSWORD},
+        ["delete-db"],
```

Comment:
what does "delete-db" command have to do with this feature? is this a stray change?

Reply:
It's because an existing database could come with existing user/password settings (including those generated by other tests and not known to this test), which would break the tests. This gets us a clean slate.